x-pack/plugins/security_solution/public/overview/components/detection_response/user_alerts_table/user_alerts_table.tsx

```diff
@@ -0,0 +1,174 @@
+
+const DETECTION_RESPONSE_USER_SEVERITY_QUERY_ID = 'vulnerableUsersBySeverityQuery';
+
+export const UserAlertsTable = React.memo(() => {
+  const { getAppUrl, navigateTo } = useNavigation();
+  const { toggleStatus, setToggleStatus } = useQueryToggle(
+    DETECTION_RESPONSE_USER_SEVERITY_QUERY_ID
+  );
+  const { data, isLoading } = useUserAlertsItems({
+    skip: !toggleStatus,
+    queryId: DETECTION_RESPONSE_USER_SEVERITY_QUERY_ID,
+  });
+
+  const columns = useMemo(
+    () => getTableColumns({ getAppUrl, navigateTo }),
+    [getAppUrl, navigateTo]
+  );
+
+  return (
+    <EuiPanel hasBorder data-test-subj="userSeverityAlertsPanel">
+      <HeaderSection
+        id={DETECTION_RESPONSE_USER_SEVERITY_QUERY_ID}
+        title={i18n.USER_ALERTS_SECTION_TITLE}
+        titleSize="s"
+        hideSubtitle
+        toggleStatus={toggleStatus}
+        toggleQuery={setToggleStatus}
+      />
+
+      {toggleStatus &&
+        (isLoading || data.length > 0 ? (
+          <>
+            <EuiBasicTable
+              data-test-subj="userAlertsTable"
+              columns={columns}
+              items={data}
+              loading={isLoading}
+              noItemsMessage={<>{'No alerts found'}</> /** TODO */}
+            />
+            <EuiSpacer size="m" />
+            <EuiButton
+              onClick={() => navigateTo({ url: getAppUrl({ deepLinkId: SecurityPageName.users }) })}
+            >
+              {i18n.VIEW_ALL_USER_ALERTS}
+            </EuiButton>
+          </>
+        ) : (
+          <EuiEmptyPrompt title={<h3>{i18n.NO_ALERTS_FOUND}</h3>} titleSize="xs" />
+        ))}
+    </EuiPanel>
+  );
+});
+
+UserAlertsTable.displayName = 'UserAlertsTable';
+
+const getTableColumns: GetTableColumns = ({ getAppUrl, navigateTo }) => [
+  {
+    field: 'userName',
+    name: i18n.USER_ALERTS_USERNAME_COLUMN,
+    truncateText: true,
+    textOnly: true,
+
+    render: (userName: string) => {
+      const url = getAppUrl({ deepLinkId: SecurityPageName.users, path: userName });
+
+      return (
+        <EuiLink
+          data-test-subj="userSeverityAlertsTable-userName"
+          href={`/app/security/users/${userName}`}
+          onClick={(ev?: React.MouseEvent) => {
+            if (ev) {
+              ev.preventDefault();
+            }
+            navigateTo({ url });
+          }}
+        >
+          {userName}
+        </EuiLink>
+      );
+    },
+  },
+  {
+    field: 'totalAlerts',
+    name: i18n.ALERTS_COLUMN,
+    render: (totalAlerts: number, { userName }) => {
+      const url = getAppUrl({ deepLinkId: SecurityPageName.users, path: userName });
+
+      return (
+        <EuiLink
+          data-test-subj="userSeverityAlertsTable-totalAlerts"
+          href={`/app/security/users/id/${userName}`}
+          onClick={(ev?: React.MouseEvent) => {
+            if (ev) {
+              ev.preventDefault();
+            }
+            navigateTo({ url });
+          }}
+        >
+          {totalAlerts}
```

Comment:
Shall we format the number with `,` here and columns below?

Reply:
resolving this one in favour of the convo above on it